README.md

```diff
@@ -31,18 +36,10 @@ The VM Restore Point feature is currently in private preview and is not meant fo
 4. Required AFECs: "Microsoft.Compute/RestorePointExcludeDisks", "Microsoft.Compute/IncrementalRestorePoints"
 
 ## Creating a VM Restore Point
-1. First Step is to create a RestorePointCollection. You can do so by using the template https://github.com/Azure/Virtual-Machine-Restore-Points/blob/main/createRestorePointCollection.json
-2. Next, you can create a restore point using the RestorePointCollection. You can do so by using the template https://github.com/Azure/Virtual-Machine-Restore-Points/blob/main/CreateRestorePoint.json
-
-2a. You can also create a restore point and exclude one or more attached disks by using the following API call:
+1. First Step is to create a RestorePointCollection. You can refer to the following API documentation on [how to create a RestorePointCollection](https://docs.microsoft.com/en-us/rest/api/compute/restore-point-collections/create-or-update#create-or-update-a-restore-point-collection.).
+2. Next, you can create a restore point using the RestorePointCollection. You can refer to the following API documentation on [how to create a RestorePoint](https://docs.microsoft.com/en-us/rest/api/compute/restore-points/create#create-a-restore-point).
+* You can also create a restore point and exclude one or more attached disks.
 
```

Comment:
[same] Why are we making changes to RestorePoints documentation?

Reply:
Restore Points team is not planning to update the GitHub repo anymore as they have the public documentation for APIs published for public preview. So added link to there public here for completeness.